src/main/java/tasks/TaskList.java

```diff
@@ -0,0 +1,187 @@
+package tasks;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+
+public final class TaskList {
```

Comment:
Should there be Javadov header comments for this class?

Reply:
added missing java documentation for TaskList, thanks for catching it!